src/morse_code.rb

```diff
@@ -0,0 +1,60 @@
+@morse_code = {
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/214766491-13f6381f-7bc9-41d8-91a8-4e6460eb1658.png)

- [ ] As you know this is a pair-programming activity but I cannot see any commit from your partner @Suzi216 side :fearful: Kindly ask your partner to make contributions in this milestone :+1:

Reply:
Ok, we will do that
Thanks @HAMMAS-SALEEM